content/software-security/comics/fighting-vulnerabilities/index.md

```diff
@@ -0,0 +1,19 @@
+---
+title: "#1 - Fighting Vulnerabilities"
```

Comment:
Maybe it can be "Inky Sea-curity/Security Adventures 1: " or something with more context? Can also include "Chainguard", etc.

Reply:
I would do `linkTitle` as something like "1 — Fighting Vulnerabilities" still

Reply:
Did not know we had an attribute for that! Cool, I can make that change